about.html

```diff
@@ -0,0 +1,183 @@
+        </section>
+      </div>
+
+      <section class="conf-info">
+        <div class="infotech">
+          <h2 class="require">
+            The Logo of the Web Dev Conference 2023 will be decided through the
+            annual Web Dev logo competition on June 20, 2023.
+          </h2>
+
+          <div class="web-dev"></div>
+          <p class="gathering">The Logo was designed by AustinTech Limited</p>
+
+          <div class="past">
+            <img src="image/logo.jpg" alt="web dev logo" />
+            <p>Web Developer Conference Logo</p>
+          </div>
+        </div>
+      </section>
+      <hr />
+
+      <section class="pictures">
+        <div class="infotech">
+          <h2 class="require">See the Past Web Dev Conference</h2>
+          <div class="web-dev"></div>
+          <p class="gathering">
+            Take a look at the last two Web DevConference which took place in
+            Abuja, Nigeria
+          </p>
+          <div class="future">
+            <div class="tomorrow">
+              <img
+                src="image/conf2021.jpg"
+                alt="web dev logo"
+                class="conf-img"
+              />
+              <div class="good">
+                <h3>2021</h3>
+                <p class="aero">Web Dev Conference 2021</p>
+              </div>
+            </div>
+
+            <div class="tomorrow">
+              <img
+                src="image/conf2022.jpg"
+                alt="web dev logo"
+                class="conf-img"
+              />
+              <div class="good">
+                <h3>2022</h3>
+                <p class="aero">Web Dev Conference 2022</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <!--supporters section-->
+
+      <section id="suporters">
+        <div class="support-container">
+          <div class="partners">
+            <h2>Partners</h2>
+            <div class="people"></div>
+            <div class="individuals">
+              <img src="image/airbnb.png" alt="airbnb" class="partner-img" />
+              <img src="image/amazon.png" alt="amazon" class="partner-img" />
+              <img src="image/coke.png" alt="coca-cola" class="partner-img" />
+              <img
+                src="image/facebook.png"
+                alt="facebook"
+                class="partner-img"
+              />
+              <img src="image/mozilla.png" alt="mozilla" class="partner-img" />
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <!--footer section-->
+
+      <section class="footer" id="footer-section">
+        <div class="materials">
+          <div class="copyright">
+            <img src="image/logoimg.jpg" alt="logo" />
+            <p>Web Developers conference</p>
+          </div>
+          <div class="year">
+            <p>&copy; 2023 Web dev conference. All rights Reserved</p>
+          </div>
+        </div>
+      </section>
+    </div>
```

Comment:
For the About page, your mobile design looks good 👍, however, we need to update the design for desktop. Remove the partners section, and set your footer to have a black background.

Figma design Desktop:
<img width="709" alt="image" src="https://github.com/Megagig/Capstone-Project/assets/31897434/a38b4b1b-c206-48ad-b092-4a6657f173ae">

Your design desktop:
<img width="1647" alt="image" src="https://github.com/Megagig/Capstone-Project/assets/31897434/b2c0f734-66ac-4128-ab9f-8c386982ba52">

Reply:
@bellom Thanks so much for this feedback. This has been corrected.